examples/using-emotion-prismjs/src/components/layout.js

```diff
@@ -19,7 +19,7 @@ const link = css`
   color: inherit;
 `
 
-class Layout extends React.Component {
+class PureLayout extends React.Component {
```

Comment:
Why the change?

Reply:
eslint wanted an explicite name (not `export default props`) and since I can't name them both `Layout` I gave this one another name.

Reply:
I guess I would've just called it BaseLayout or something! Pure implies (to me!) something like a PureComponent i.e. a component that doesn't re-render when its props are shallowly equal.

Reply:
And ai chose Pure because it’s without GraphQL 😀 But I don’t care really

Reply:
Yeah, not a big deal either way, _especially_ because this is an example. Let's just merge :) Thank you!